src/bond.py

```diff
@@ -63,3 +63,104 @@ def __init__(self):
+    the simulation state with:
+
+    .. math::
+        :nowrap:
+
+        \begin{eqnarray*}
+        U(r) &= k (r - \Delta - r_0 - b_1)(r - \Delta - r_0 - b_2)(r - \Delta - r_0)^2
+                  + U_0 + U_{\rm WCA}(r), & r < r_0 + \Delta \\
+             &= U_0, & r \ge r_0 + \Delta
+        \end{eqnarray*}
+
+    where :math:`\vec{r}` is the vector pointing from one particle to the other in the
+    bond. The potential :math:`U_{\rm WCA}(r)` is given by:
+
+    .. math::
+        :nowrap:
+
+        \begin{eqnarray*}
+        U_{\rm WCA}(r) &= 4 \varepsilon \left[ \left( \frac{\sigma}{r-\Delta} \\
+                                                                            \right)^{12}
+                             - \left( \frac{\sigma}{r-\Delta} \right)^{6} \right] \\
+                             + \varepsilon,
+                                          & r < 2^{\frac{1}{6}}\sigma + \Delta  \\
+                           &= 0,          & r \ge 2^{\frac{1}{6}}\sigma + \Delta
+        \end{eqnarray*}
+
+
+    Attributes:
+        params (TypeParameter[``bond type``, dict]):
+            The parameters of the quartic bonds for each particle type.
+            The dictionary has the following keys:
+
+            * ``k`` (`float`, **required**) - quartic attractive force strength
+              :math:`[\mathrm{energy/length^4}]`.
+
+            * ``r_0`` (`float`, **required**) - Location of the quartic potential
+              cutoff :math:`r_0`. Intended to be larger than the WCA cutoff,
+              :math:`2^{\frac{1}{6}}\sigma`. When true,
+              :math:`U(r_0) = U_{0} + U_{\rm WCA}(r_0)`
+              :math:`[\mathrm{length}]`.
+
+            * ``b_1`` (`float`, **required**) - First quartic potential fitting
+              parameter :math:`b_1` :math:`[\mathrm{length}]`.
+
+            * ``b_2`` (`float`, **required**) - Second quartic potential fitting
+              parameter :math:`b_2` :math:`[\mathrm{length}]`.
+
+            * ``U_0`` (`float`, **required**) - Quartic potential energy barrier height
+              :math:`U_0` at :math:`r_0` when :math:`r_0 > 2^{\frac{1}{6}}\sigma`
+              :math:`[\mathrm{energy}]`.
+
+            * ``epsilon`` (`float`, **required**) - Repulsive WCA interaction energy
+              :math:`\varepsilon` :math:`[\mathrm{energy}]`.
+
+            * ``sigma`` (`float`, **required**) - Repulsive WCA interaction size
+              :math:`\sigma` :math:`[\mathrm{length}]`.
+
+            * ``Delta`` (`float`, **optional**) - Shift :math:`\Delta`,
+              defaults to zero :math:`[\mathrm{length}]`.
+
+    Examples::
+
+        quartic = azplugins.bond.Quartic()
+        quartic.params['A-A'] = dict(k=1434.3, r_0=1.5, b_1=-0.7589, b_2=0.0,
+                                    U_0=67.2234, sigma=1, epsilon=1)
+        quartic.params['B-B'] = dict(k=1434.3, r_0=1.5, b_1=-0.7589, b_2=0.0,
+                                    U_0=67.2234, sigma=1, epsilon=1, delta=0.2)
```

Comment:
Is delta really zero and 0.2 in the paper below? It seems unlikely. If not, please only include the example from the paper. Please also see here for how to group text and code together in an example using the `rubric` directive:

https://github.com/glotzerlab/hoomd-blue/blob/dc715ea70b65a3330d65c6b6bc51de09b8d1742f/hoomd/mpcd/collide.py#L154-L163